docs/model-dev-guide/api-guides/batch-processing/batch-process-api-ug.rst

```diff
@@ -151,7 +162,37 @@ example.
 
            self.output = []
 
-Step 2: Initialize the Dataset
+(Optional) Step 2: Link the Run to a Checkpoint or Model Version
+================================================================
+
+You can optionally link your batch inference run with the
+:class:`~determined.experimental.checkpoint.Checkpoint` or
+:class:`~determined.experimental.model.ModelVersion` used in the run to aggregate custom metrics on
+that object for analysis later.
+
+A full example of this can be found in the `CIFAR10 Pytorch Example
+<https://github.com/determined-ai/determined/tree/main/examples/computer_vision/cifar10_pytorch>`__
+in `inference_example.py`.
+
+Connect the :class:`~determined.experimental.checkpoint.Checkpoint` or
+:class:`~determined.experimental.model.ModelVersion` to the inference run.
+
+.. code:: python
+
+   # Checkpoint
+   ckpt = client.get_checkpoint(hparams.get("checkpoint_uuid"))
+   core_context.utils.report_task_using_checkpoint(ckpt)
+
+   # Or Model Version
+   model = client.get_model(hparams.get("model_name"))
+   model_version = model.get_version(hparams.get("model_version"))
+   self.core_context.utils.report_task_using_model_version(model_version)
+
+Any metrics that you report within the trial will now be accessible to any query to `.get_metrics()`
+on the :class:`~determined.experimental.checkpoint.Checkpoint` or
+:class:`~determined.experimental.model.ModelVersion` used.
+
```

Comment:
[double backticks]

The :class:`~determined.experimental.checkpoint.Checkpoint` and
:class:`~determined.experimental.model.ModelVersion` used are now available to any query via ``.get_metrics()``.

Reply:
added